subgraphs/_reference_/src/sdk/protocols/bridge/protocol.ts

```diff
@@ -146,10 +146,10 @@ export class Bridge {
 
   /**
    *
-   * @returns {ethereum.Event} the event currently being handled.
+   * @returns {ethereum.Block} the block currently being handled.
    */
-  getCurrentEvent(): ethereum.Event {
-    return this.event;
+  getCurrentBlock(): ethereum.Block {
```

Comment:
And then the same here, make it return `CustomEventType` for whomever needs to have the raw event.